src/services/API/endpoints.js

```diff
@@ -1 +1,2 @@
 export const checkUserEndpoint = "/checkForUser"
+export const getProgramDataEndpoint = "/getProgramData"
```

Comment:
Maybe not valid right now but will be soon, we have to think this over as well.

If other people can also use the auth layer, they might have a different place where they're storing their programData (or groupData) so we need to provide the support for hooking up their data store to get the program data file. @pritamps